appdatabase/migrations/sql/1625872445_user_status.up.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE settings ADD COLUMN current_user_status BLOB;
+ALTER TABLE settings ADD COLUMN send_status_updates BOOLEAN DEFAULT TRUE;
+
```

Comment:
You will need also to set the existing rows to `TRUE`, as otherwise they will be `nil`/`false` (existing rows won't be impacted by defaulting a new field)